tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -2333,28 +2291,12 @@ public function userHasBeenDeleted(string $user):void {
 			if ($this->isTestingWithLdap() && \in_array($user, $this->ldapCreatedUsers)) {
 				$this->deleteLdapUser($user);
 			} else {
-				$this->deleteTheUserUsingTheProvisioningApi($user);
+				$this->deleteUser($user);
```

Comment:
check that deletion was successful
```suggestion
				$this->deleteUser($user);
				$this->theHttpStatusCodeShouldBe(204, "", $response);
```